python-package/README.md

```diff
@@ -72,12 +72,12 @@ running on a confidential ACI instance with the corresponding attestation
 document issuer, and submit an HTTP request:
 
 ```python
-from atls import AttestedHTTPSConnection, AttestedTLSContext
+from atls import HTTPAConnection, ATLSContext
```

Comment:
Would be great to add the `httpa` example here.

Reply:
Added.